modules/tinymce/src/plugins/template/main/ts/ui/Dialog.ts

```diff
@@ -13,6 +13,7 @@ import * as Utils from '../core/Utils';
 type UpdateDialogCallback = (dialogApi: Dialog.DialogInstanceApi<DialogData>, template: InternalTemplate, previewHtml: string) => void;
 
 const getPreviewContent = (editor: Editor, html: string): string => {
+  const previewHtml = Cell(Utils.parseAndSerialize(editor, html));
```

Comment:
I don't think you need to have a cell here since it's all local mutation inside that function. I think cells are a great way of say tracking mutation though multiple functions and modules. You instantly see that something is mutable if you wrap it in a cell type. In this case I think it would be easier to just use a let expression.

Reply:
That's good to know, thanks for the explanation.